packages/@icon-magic/generate/src/plugins/svg-light-dark.ts

```diff
@@ -0,0 +1,179 @@
+/*
+ * This plugin will create a master svg with two child svg's.
+ * One dark and one light.
+ * The children svgs will have a `display` attribute that can be used to set
+ * which of the two svg's will be visible.
+ */
+
+import {
+  Flavor,
+  GeneratePlugin,
+  Icon
+} from '@icon-magic/icon-models';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Svgo from 'svgo';
+import { DOMImplementation, DOMParser, XMLSerializer } from 'xmldom';
+
+const serializeToString = new XMLSerializer().serializeToString;
+
+export interface SvgLightDarkOptions {
+  lightToken?: string;
+  darkToken?: string;
+}
+
+export const svgLightDark: GeneratePlugin = {
+  name: 'svg-light-dark',
+  fn: async (
+    flavor: Flavor,
+    icon: Icon,
+    params: SvgLightDarkOptions = {},
+  ): Promise<Flavor> => {
+    // Get the output directory
+    const outputPath = icon.getIconOutputPath();
+    const imageSet = flavor.imageset;
+    const colorScheme = flavor.colorScheme;
+
+    /*
+    * If `imageSet` exists and the `colorScheme` equals `dark, run the plugin.
+    * Plugin will skip the `light` flavors because their data can be collected through the `dark` flavor
+    */
+    if (imageSet && colorScheme === 'dark') {
+      // light flavor data
+      const lightFlavor = icon.flavors.get(imageSet);
+
+      // if light flavor doesn't exist, exit out of plugin
+      if (!lightFlavor) {
+        return flavor;
+      } else {
+        const darkFlavorContents = (await flavor.getContents()) as string; // .svg asset's getContents() returns the svg as a string
+        const lightFlavorContents  = (await lightFlavor.getContents()) as string;
+
+        const { lightToken, darkToken} = params;
+
+        const lightSvgo = new Svgo({
+          plugins: [
+            {
+              addAttributesToSVGElement: {
+                attributes: [{display: lightToken || "var(--svg-light-display)"}]
+              }
+            },
+            {
+              removeViewBox: false
+            }
+          ],
+          js2svg: { pretty: true, indent: 2 }
+        });
+
+        const darkSvgo = new Svgo({
+          plugins: [
+            {
+              addAttributesToSVGElement: {
+                attributes: [{display: darkToken || "var(--svg-dark-display)"}]
```

Comment:
```suggestion
                attributes: [{display: `var(${darkToken})` || "var(--svg-dark-display)"}]
```

Reply:
the way he sets it in the iconrc, it already has `var`

Reply:
Yeah I thought it was fine like that but we can change in to just be the token name?